lib/iris/fileformats/grib/__init__.py

```diff
@@ -636,6 +636,20 @@ def _get_verification_date(self):
         # Return validity_time = (reference_time + start_offset*time_unit).
         return reference_date_time + interval_delta
 
+    @property
+    def bmdi(self):
+        # Not sure of any cases where GRIB provides a fill value.
+        # Default for fill value is None.
+        return None
```

Comment:
@lbdreyer I'm assuming that `bmdi` is used by the rules?

We need to verify GRIBs relationship with `fill_value` ...